react_on_rails_pro/spec/dummy/e2e-tests/fixture.ts

```
@@ -0,0 +1,129 @@
+}
+
+export type RedisReceiverControllerFixture = {
+  sendRedisValue: (key: string, value: unknown) => Promise<void>;
+  sendRedisItemValue: (itemIndex: number, value: unknown) => Promise<void>;
+  matchPageSnapshot: (snapshotPath: string) => Promise<void>;
+  waitForConsoleMessage: (msg: string) => Promise<void>;
+  getNetworkRequests: (requestUrlPattern: RegExp) => Promise<Request[]>;
+}
+
+const redisControlledTest = base.extend<RedisRequestIdFixture, RedisClientFixture>({
+  redisClient: [async ({}, use, workerInfo) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)